lib/wise_homex/models/household.ex

```diff
@@ -13,6 +13,7 @@ defmodule WiseHomex.Household do
     has_many :settlement_values, WiseHomex.SettlementValue
     has_many :unsuccessful_visits, WiseHomex.UnsuccessfulVisit
     has_one :ready_installation, WiseHomex.ReadyInstallation
+    has_one :ds_energy_location, WiseHomex.DSEnergyLocation
```

Comment:
This relation is not exposed in the API

Reply:
Good catch. Sorry about that. Now exposed in the API :sunny: